### PR TITLE
Fixappinsights

### PR DESCRIPTION
**Pull request by colindembovsky**
Fixing various screenshots and other small things - mostly to do with updates to VSTS since this lab was initially created. Also fixing the ARM template (the region has changed for AppInsights)

### CODE REVIEW

File: docs/HOL-Continuous_Deployment/README.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ button next to "Azure Resource Group Deployment" to add the task. Close the "Tas
 		-CdnStorageContainerNameForDev $(ContainerName)-dev
 		-CdnStorageAccountNameForStaging $(StorageAccountName)-stage
 		-CdnStorageContainerNameForStaging $(ContainerName)-stage
-		-PartsUnlimitedServerAdminLoginPassword (ConvertTo-SecureString -String '$(AdminPassword)' -AsPlainText -Force)
+		-PartsUnlimitedServerAdminLoginPassword (ConvertTo-SecureString -String '$(AdminPw)' -AsPlainText -Force)
```
Comment (julienstroheker) [inline docs/HOL-Continuous_Deployment/README.md]: Why are you changing the name of this variable ?

File: docs/HOL-Continuous_Deployment/README.md [hunk 2]
```diff
@@ -220,7 +220,7 @@ button next to "Azure Resource Group Deployment" to add the task. Close the "Tas
 	* **HostingPlan** - Name of the hosting plan for the website
 	* **StorageAccountName** - Storage account name prefix. Will have `-dev` or `-stage` added for dev/staging
 	* **ContainerName** - Container name prefix. Will have `-dev` or `-stage` added for dev/staging
-	* **AdminPassword** - Admin password for production database server
+	* **AdminPw** - Admin password for production database server
```
Comment (julienstroheker) [inline docs/HOL-Continuous_Deployment/README.md]: Same thing, Why are you changing the name of this variable ?

File: docs/HOL-Continuous_Integration/README.md [hunk 3]
```diff
@@ -33,55 +33,58 @@ If you haven't already, create a new team project in your Visual Studio Team Ser
...
-Now you can push the code, including history, to VSTS:
+```
+git clone <vsts-git-url> HOL
+```
 
-	git push -u origin --all
-Congratulations, your code should now be in VSTS!
+After a few seconds of downloading, all of the code should now be on your local machine.
 
 ### II. Create Continuous Integration Build
```
Comment (julienstroheker) [inline docs/HOL-Continuous_Integration/README.md]: Please remove those changes from this PR

Comment (julienstroheker) [inline docs/HOL-Continuous_Integration/README.md]: And also all the images bellow

**Comment by dtzar** [comment]
@julienstroheker this fixes a core deployment problem #118 so we need to get this merged.  I just tested deployment template works now and reviewed the content.

File: docs/HOL-Continuous_Integration/README.md (hunk 3 above)
Comment (dtzar) [inline docs/HOL-Continuous_Integration/README.md]: why are you asking to remove those additional images?

File: docs/HOL-Continuous_Deployment/README.md (hunk 1 above)
Comment (dtzar) [inline docs/HOL-Continuous_Deployment/README.md]: agreed - this makes no sense to change the name.  @colindembovsky can you please just revert it?

**Comment by julienstroheker** [comment]
Are we talking about the App Insights Location ? If yes, we already fixed it here #115

**Pull request closed without merging**